x-pack/platform/plugins/shared/onechat/server/services/runner/utils/get_default_connector.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { InferenceConnector, InferenceConnectorType } from '@kbn/inference-common';
+
+/**
+ * Naive utility function to consistently return the "best" default connector for onechat features.
+ *
+ * For now, the logic is, by order of priority:
```

Comment:
and in future this would be biased towards the EIS "rainbow sprinkles" connector, above "any inference connector"